src/styles/content.scss

```diff
@@ -57,9 +60,177 @@ table.os-raise-standardtable {
   }
 }
 
+%os-raise-basictable {
+  /* border-collapse: separate paired with border-spacing: 0 allows for a border-radius
+     to be applied to the entire table (border-collapse: collapse will ignore any border
+     radius applied to the table)
+  */
+  border: 2px solid $raise-table-color;
+  border-collapse: separate;
+  border-spacing: 0;
+  border-radius: 5px;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
+    font-weight: 600;
+    text-align: left;
+  }
+
+  thead {
+    background-color: $raise-table-color;
+  }
+
+  th,
+  td {
+    text-align: center;
+    padding: 16px;
+  }
+
+  th {
+    font-size: 1.2rem;
+    font-weight: 700;
+    color: #fff;
+  }
+
+  td {
+    font-size: 1.067rem;
+    color: $raise-light-black;
+    background-color: #fff;
+    border-left: 1px solid $raise-table-color;
+    border-top: 1px solid $raise-table-color;
+  }
+}
+
+table.os-raise-doubleheadertable {
+  table-layout: fixed;
+  width: 100%;
+
+  caption {
+    color: $raise-light-black;
+    font-size: 1.6rem;
```

Comment:
Same comment as above as it relates to possibly correcting `1.6rem` to be `1.5rem`.